src/Microsoft.Azure.WebJobs.ServiceBus/Constants.cs

```diff
@@ -5,7 +5,7 @@
 
 namespace Microsoft.Azure.WebJobs.ServiceBus
 {
-    internal static class Constants
+    public static class Constants
```

Comment:
Tests needed this, and as a rule of thumb, we've been avoiding just relying on InternalsVisibleTo. I can make this internal, but it would be useful to consumers.